gradle-jdks-groovy/src/main/groovy/com/palantir/gradle/jdks/ConfigureJdksIdeaPluginXml.groovy

```diff
@@ -39,7 +38,9 @@ class ConfigureJdksIdeaPluginXml {
             if (!createIfAbsent) {
                 return;
             }
-            rootNode = new Node(null, "project", ImmutableMap.of("version", "4"));
+            // Using Map rather than ImmutableMap to avoid a Guava dependency.
+            Map<String, String> projectNodeAttributes = Map.of("version", "4");
```

Comment:
I could also add a Guava dependency to `gradle-jdks-groovy` instead of using `Map.of`. I saw this class gets called from `ToolchainsPlugin`, which already had a Guava dep.

```
- ToolchainsPlugin
  - PalantirGradleJdksIdeaPlugin
    - ConfigureJdksIdeaPluginXml
```

Do we need to keep `gradle-jdks-groovy` dependency-free?

Reply:
Better to use `Map.of` than require Guava anyway, bonus to avoid the dependency.